src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/BeneficialOwnerAdditionService.java

```diff
@@ -89,6 +89,10 @@ private IndividualBeneficialOwnerAddition getIndividualBeneficialOwnerAddition(
                         addressDtoToAddress(serviceAddress),
                         natureOfControls,
                         isOnSanctionsList);
+        
+        if (!bo.getTrustIds().isEmpty()) {
+                individualBeneficialOwnerAddition.setTrustIds(bo.getTrustIds());
```

Comment:
Indentation?

Reply:
Indentation fixed, unsure what you mean by the question

Reply:
Consistent now but I was questioning the indentation of the whole file. And commenting that we could just always set here but ok.